src/Install/DatabaseConfig.php

```diff
@@ -47,7 +47,7 @@ public function toArray()
             'collation' => 'utf8mb4_unicode_ci',
             'prefix'    => $this->prefix,
             'strict'    => false,
-            'engine'    => 'InnoDB',
+            'engine'    => NULL,
```

Comment:
```suggestion
            'engine'    => null,
```

Convention to use lower case.